opm/material/densead/Evaluation.hpp

```diff
@@ -538,9 +538,7 @@ Evaluation<ValueType, numVars> operator+(const RhsValueType& a, const Evaluation
 template <class RhsValueType, class ValueType, int numVars>
 Evaluation<ValueType, numVars> operator-(const RhsValueType& a, const Evaluation<ValueType, numVars>& b)
 {
-    Evaluation<ValueType, numVars> result(a);
-    result -= b;
-    return result;
+    return -(b - a);
```

Comment:
This change looks odd to me: what is the benefit of this change?

Reply:
I think ` Evaluation<ValueType, numVars> result(a);` does not work for `DynamicEvaluation`. 

With this change we reuse ` Evaluation operator-(const RhsValueType& other) const`.

Reply:
But it is slower. Maybe we should look into other options.